ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/BraveTranslateScriptHandler.swift

```diff
@@ -71,6 +71,12 @@ class BraveTranslateScriptHandler: NSObject, TabContentScript {
   ) {
     // Setup
 
+    if !FeatureList.kBraveTranslateFeature.enabled {
+      Logger.module.debug("Translation Feature Disabled")
+      replyHandler(nil, BraveTranslateError.translateDisabled.rawValue)
+      return
+    }
+
```

Comment:
This shouldn't be necessary since you shouldn't even be adding the script handler if its disabled